docs/dev/android/setup-android-build-environment.md

```diff
@@ -17,19 +17,24 @@ At the end of this process you should have the following environment variables s
 
 - `ANDROID_HOME`
 - `NDK_HOME`
+- `JAVA_HOME`
 
 The easiest way to install all the dependencies (and automatically
 handle updates), is by using [Android Studio](https://developer.android.com/studio/index.html).
-Once this is installed, the following dependencies can be installed through it:
+Once this is installed, it must be run and the glean project opened to complete initial setup.
+If Android Studio asks you to upgrade the verison of gradle, decline.
```

Comment:
I wish there was a way to just tell Android Studio to not do this for everyone using the project, but Google is turning up blank.  In practice, gradle isn't very stable and upgrading it requires a lot of changes, so I don't know why it's so aggressive about this.  Anyway, `</rant>`.  This comment is good to have here.